docker/db/mysql/init/create_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE BOARD
```

Comment:
```suggestion
CREATE TABLE BOARDS
```
테이블 명은 보통 복수로 관리할 것 같아요!

Reply:
테이블마다 id는 필요없을까요?

Reply:
지금 당장에는 하나의 게임만을 저장하고, 때문에 하나의 보드 정보만을 저장하게 될 것 같아서 `BOARD`라는 네이밍을 했습니다. 동일하게 하나의 게임만을 저장하기 때문에 `id` 값 역시 다음 단계에서 추가를 하는 것이 맞을 것 같다고 생각을 했고요.

그러나, 이렇게 구현을 하니 특히 `GAME` 테이블이 1개의 컬럼과 1개의 로우만을 가지게 되더라고요... 😅 
조금은 이후의 변경을 염두에 두는 방법이겠지만, 복수형을 사용하고 `id`도 추가하는 방향으로 변경해 볼게요!